service/project.go

```diff
@@ -76,15 +92,16 @@ func (s *ProjectService) Delete(ctx context.Context, projectID uuid.UUID, authUs
 		return err
 	}
 
-	_, err := s.Get(ctx, projectID, authUserID)
-	if err != nil {
+	if exists, err := s.Exists(ctx, projectID, authUserID); err != nil {
 		return err
+	} else if !exists {
```

Comment:
Still, there's some friction when reading this. :D

An `if` statement ending not with `else`. Meh.. 😕 

Consider flattening it a bit. I don't mind additional lines of code.

Reply:
flattening, making it more simple and obvious...

Reply:
I am going to drop the second commit, and we can discuss the best solution in issue #23, as this seems to have been the catalyst for discussions about moving the environment to a separate service